src/sentry.expo.ts

```diff
@@ -78,8 +78,16 @@ function isPublishedExpoUrl(url: string) {
   return url.includes('https://d1wp6m56sqw74a.cloudfront.net');
 }
 
+/**
+ * Filenames such as 
+ * `/data/user/0/host.exp.exponent/files/.expo-internal/bundle-AD3BEBE4AD9CFD8AF700EE807D2762758B2A1DA0D7FAA79A285D9BF25CC3A361`
+ */
+function isLocalBundleFile(url: string) {
+  return !url.endsWith('.js') && url !== '[native code]'
```

Comment:
i would've thought this check for `!url.endsWith('.js')` would break iOS over-the-air updates in managed workflow. The bundle url for those is 
`https://d1wp6m56sqw74a.cloudfront.net/%40charliecruzan%2Fsdk38%2F1.0.0%2F6d061333ad1a49ace5de400a076fa40a-39.0.0-ios.js` (as an example), see the breadcrumb console in this error for full error details including stack-> https://sentry.io/share/issue/8e515fa58fcc47d08a9f278915d2a589/

it seems to still map it properly though 🤔 (although on an [earlier test it did not](https://sentry.io/share/issue/1c0ad82d5e1142d79aa96315b70d910e/), but maybe i was missing something)

Reply:
It should map properly I think because it would get hit by `isPublishedUrl` first. 
https://github.com/expo/sentry-expo/blob/b7cbf01047200616c938bbdf2789c84d9759bf90/src/sentry.expo.ts#L90

Reply:
oh you're right, I completely glossed over that 😅